dts/arm/nordic/nrf52805.dtsi

```diff
@@ -320,3 +320,8 @@
 &nvic {
 	arm,num-irq-priority-bits = <3>;
 };
+
+&systick {
+	/* Use RTC for system clock, instead of SysTick. */
+	status = "disabled";
+};
```

Comment:
systick change in `dts: Move nrf_common.h to the common directory`, should be split.

Reply:
This is just moved here from `nrf_common.dtsi`, as this is an ARM specific thing so it can no longer stay there. I added a related note in the commit message.